docs/code-quality/c6504.md

```diff
@@ -16,10 +16,9 @@ ms.workload:
 # C6504
 warning C6504: invalid annotation: property may only be used on values of pointer, pointer-to-member, or array type
 
- This warning indicates the use of a property on an incompatible data type. For more information about data types supported by properties, see [Annotation Properties](using-sal-annotations-to-reduce-c-cpp-code-defects.md).
+ This warning indicates the use of a pointer specific sal annotation on a non-pointer data type. For more information about what data types are supported by properties, see [Annotation Properties].
```

Comment:
missing the link?

Reply:
This works, it references the link used later in the document.